### PR TITLE
Create Docker and Compose files for active development

### PR DESCRIPTION
**Pull request by kabilar**
- User forks and clones `element-lab`, `element-animal`, `element-session`, `element-array-ephys`, and `workflow-array-ephys` packages to their host machine.
- These packages are then installed within the Docker image.
- These packages can be modified on the host machine, and the updates will be reflected within the Docker container.
- In the future we should possibly consolidate these sets of Docker and Compose files.

### CODE REVIEW

**Comment by kabilar** [comment]
Hi @CBroz1 @tdincer @davidgodinez @GeetikaSi, this pull request includes the updates we discussed this morning.  Thanks.

**Review by ttngu207 — APPROVED** [review]

**Comment by CBroz1** [comment]
I really like this as a standard moving forward. My only reservation is how many files this adds to the root workflow directory. We're asking the workflows to do double duty: (a) run integration tests/function as dev environment, (b) instruct users on how to use our tools. Can we migrate the docker files to a subdirectory to provide novice users with a clearer delineation between dev and didactic materials?

**Comment by kabilar** [comment]
Hi @CBroz1, that's a good idea.  I am looking for best practices and examples on how to organize these files.

File: Dockerfile.test [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN git clone https://github.com/datajoint/workflow-array-ephys.git .
 # COPY --chown=anaconda:anaconda ./element-array-ephys /main/element-array-ephys
 # RUN pip install /main/element-array-ephys
 # COPY --chown=anaconda:anaconda ./workflow-array-ephys /main/workflow-array-ephys
-# RUN rm /main/workflow-array-ephys/dj_local_conf.json
+# RUN rm -f /main/workflow-array-ephys/dj_local_conf.json
```
Comment (CBroz1) [inline Dockerfile.test:23]: Does user anaconda have rm permissions in all directories? When I open a CLI for `datajoint/djlab:py3.8-debian`, I don't have permission for `rm` or `mv`

**Pull request merged**